cartography/intel/aws/ec2.py

```diff
@@ -72,6 +84,32 @@ def get_ec2_vpcs(session, region):
     return client.describe_vpcs()
 
 
+def load_ec2_key_pairs(session, data, region, current_aws_account_id, aws_update_tag):
+    ingest_key_pair = """
+    MERGE (keypair:KeyPair:EC2KeyPair{keyname: {KeyName}, region: {Region}})
```

Comment:
[schema standard] we recommend the use of a unique identifying attribute 'id' to help standardize on what to MERGE on and index.

Reply:
The id could be inferred by keyname-region for example.

Reply:
To elaborate on Sacha's comment, we recommend doing something like
```
MERGE (keypair:KeyPair:EC2KeyPair{id: {KeyName}, keyname: {KeyName}, region: {Region}})
# assuming of course that {KeyName} uniquely identifies a KeyPair in all of AWS
# For this reason we like ARNs if possible
```